docs/content/en/docs/installation/prerequisites/_index.md

```diff
@@ -10,3 +10,14 @@ Make sure you have the following:
 
 - To use the operator, you need admin access to a Kubernetes cluster.
 - For playground usage you can use [kind](https://kind.sigs.k8s.io/) for example.
+- For productive usage [Helm](https://helm.sh) is recommended.
+
+## [Cert Manager](https://cert-manager.io/)
+
+The Aiven Operator for Kubernetes uses `cert-manager` to configure the [service reference](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#service-reference) of our webhooks.
+
+
+Please follow the [installation instructions](https://cert-manager.io/docs/installation/helm/) on their website.
+
+> Note: this is not required in the helm installation if you select to disable webhooks, but that is not recommended
```

Comment:
Two little things here:
- Capitalize the word "Helm"
- Can you tell the user why this is not recommended?

Reply:
:heavy_check_mark: